vkrunner/vr-box.c

```diff
@@ -285,11 +297,15 @@ compare_value(enum vr_box_comparison comparison,
         case VR_BOX_BASE_TYPE_FLOAT:
                 return compare_double(comparison,
                                       *(const float *) a,
-                                      *(const float *) b);
+                                      *(const float *) b,
+                                      tolerance,
+                                      percent);
         case VR_BOX_BASE_TYPE_DOUBLE:
                 return compare_double(comparison,
                                       *(const double *) a,
-                                      *(const double *) b);
+                                      *(const float *) b,
```

Comment:
This looks like a cut-and-paste mistake. I guess this should be `*(const double *)`?